lib/ts/index.ts

```diff
@@ -91,6 +92,52 @@ export default class AuthHttpRequest {
             userContext: getNormalisedUserContext(input === undefined ? undefined : input.userContext)
         });
     };
+
+    static getInvalidClaimsFromResponse = async function(input: {
+        response: { data: any } | Response;
+        userContext?: any;
+    }): Promise<ClaimValidationError[]> {
+        return AuthHttpRequestFetch.recipeImpl.getInvalidClaimsFromResponse({
+            response: input.response,
+            userContext: getNormalisedUserContext(input.userContext)
+        });
+    };
+
+    static validateClaims = (
+        overrideGlobalClaimValidators?: (
+            globalClaimValidators: SessionClaimValidator[],
+            userContext: any
+        ) => SessionClaimValidator[],
+        userContext?: any
+    ): Promise<ClaimValidationError[]> | ClaimValidationError[] => {
+        const normalisedUserContext = getNormalisedUserContext(userContext);
+        const claimValidatorsAddedByOtherRecipes = AuthHttpRequest.getClaimValidatorsAddedByOtherRecipes();
+        const globalClaimValidators = AuthHttpRequestFetch.recipeImpl.getGlobalClaimValidators({
+            claimValidatorsAddedByOtherRecipes,
+            userContext: normalisedUserContext
+        });
+        const claimValidators =
+            overrideGlobalClaimValidators !== undefined
+                ? overrideGlobalClaimValidators(globalClaimValidators, normalisedUserContext)
+                : globalClaimValidators;
+
+        if (claimValidators.length === 0) {
+            return [];
+        }
+
+        return AuthHttpRequestFetch.recipeImpl.validateClaims({
+            claimValidators,
+            userContext: getNormalisedUserContext(userContext)
+        });
+    };
+
+    static addClaimValidatorFromOtherRecipe = (builder: SessionClaimValidator) => {
+        AuthHttpRequest.claimValidatorsAddedByOtherRecipes.push(builder);
+    };
+
+    static getClaimValidatorsAddedByOtherRecipes = (): SessionClaimValidator[] => {
+        return AuthHttpRequest.claimValidatorsAddedByOtherRecipes;
+    };
```

Comment:
You can export these through supertokens-website/utils, and on web-js side, via supertokens-web-js/recipe/session/utils (if required at all)